homeassistant/components/roku/browse_media.py

```diff
@@ -148,7 +159,5 @@ def library_payload(coordinator):
         for child in library_info.children
     ):
         library_info.children_media_class = MEDIA_CLASS_CHANNEL
-    else:
```

Comment:
@MartinHjelmare do you think i should of kept this one. Still wrapping my head around the structure

Reply:
For items with children it will default children_media_class to directory:
https://github.com/home-assistant/core/blob/6b317ced17efade42b2d5d821cadcf12ae8513a8/homeassistant/components/media_player/__init__.py#L1087-L1096